open-metadata-implementation/adapters/open-connectors/repository-services-connectors/open-metadata-collection-store-connectors/graph-repository-connector/docs/overview-of-the-graph-repository-connector.md

```diff
@@ -472,7 +472,7 @@ main you won’t, and you will need to perform a property-based search, such as:
 g.V().has('veReferenceablexqualifiedName','csv-asset-discovery-service-GovernanceServiceProvider')
 
 However, at the time of writing I cannot do that because the JanusGraph 0.5.2 assembly I am using has an older version of Lucene than 
-that used in the Egeria 3.13-SNAPSHOT build, so I am unable to use the index. I would need to build a JanusGraph assembly with a later 
+that used in the Egeria 2.6-SNAPSHOT build, so I am unable to use the index. I would need to build a JanusGraph assembly with a later 
```

Comment:
Seems an odd number to go back to?

Reply:
(but it's minor/optional) - this connector will go out soon.?

Reply:
The text description refers to a limitation for that specific build of Egeria and old JanusGraph version. This should not have been changed in the first place since so I decided to put back the original text. Anyway I agree will go in separate repo soon.